integration-tests/scripts/install.sh

```diff
@@ -201,34 +201,62 @@ tpa_integration() {
   fi
 }
 
+artifactory_integration() {
+  if [[ "${registry_config}" == "artifactory" ]]; then
+    echo "[INFO] Configure Artifactory integration into RHTAP"
+    # disable Quay installation
+    yq e '.rhtapCLI.features.redHatQuay.enabled = false' -i "${config_file}"
```

Comment:
what happens when quay is enabled then artifactory and nexus is also installed?

Reply:
if you just run the install.sh to install rhtap, this case never happens. If we need the case you mention, then we need to consider if we need to support rhtap with mulitiple Container Registries, and SCM, etc.